base/sysinfo.jl

```diff
@@ -640,5 +642,18 @@ function username()
     isempty(pw.username) && Base.uv_error("username", Base.UV_ENOENT)
     return pw.username
 end
+"""
```

Comment:
add a blank linke before docstring
```suggestion

"""
```